bin/limitd

```diff
@@ -37,6 +39,18 @@ if (config.configFile) {
   }
 }
 
+if (config.parseEnv) {
```

Comment:
Yes, do you want to parse always the Environment and remove the command line parameter? I can do it that way.